File: app/src/main/java/com/castelijns/mmdemo/albums/AlbumsContract.java
package com.castelijns.mmdemo.albums;

import com.castelijns.mmdemo.app.BasePresenter;
import com.castelijns.mmdemo.app.BaseView;
import com.castelijns.mmdemo.models.Album;

import java.util.List;

public interface AlbumsContract {

    interface View extends BaseView {
        void showAlbums(List<Album> albums);
        void showAlbumCount(int count);
    }

    interface Presenter extends BasePresenter {

    }
}

File: app/src/main/kotlin/com/castelijns/mmdemo/albums/AlbumsContract.kt
package com.castelijns.mmdemo.albums

import com.castelijns.mmdemo.app.BaseListView
import com.castelijns.mmdemo.models.Album

interface AlbumsContract {

    interface View : BaseListView {
        fun showAlbums(albums: List<Album>)
        fun showAlbumCount(count: Int)
    }

}
